### ISSUE TITLE
Create an summarize function for category (text) field in the Pivot tables

### ISSUE THREAD

**Issue by bas-ter** [metabase/metabase#18024]
Pivot tables are great for summing numbers, but currently, it is absolutely impossible to display text as an aggregated value in the Pivot table. But in different tasks, it is often necessary to display a text in the value area of a pivot table. 

Create an aggregate using a category (text) field.

As an example, other Bi solutions provide this functionality in several ways:
In the Microsoft Power Bi, you can also [aggregate a non-numeric field](https://docs.microsoft.com/en-us/power-bi/create-reports/service-aggregates#create-an-aggregate-using-a-category-text-field). For example, if you have a product name field, you can add it as a value and then set it to First, or Last. The Values well is typically used for numeric fields. Power BI recognizes that this field is a text field, sets the aggregate to Do not summarize, and presents you with a single-column table.

[In Excel, it is possible to use DAX with CONCATENATEX](https://sfmagazine.com/post-entry/july-2018-excel-reporting-text-in-a-pivot-table/)

It is a very important feature for reports with text values to be presented in the Pivot view.

**Comment by flamber** [comment]
Hi @bas-ter 
It sounds like you are looking for a Custom Expression similar to `STRING_AGG` on databases, which isn't specific to Pivot Table.
But the links seems to be requesting advanced Pivot functionality - almost the full editor.

**Comment by bas-ter** [comment]
Yes, it should be like a pivot table level aggregation that can work with text. For example, if multiple values appear at the intersection of columns and rows, show only the first or last value in alphabetical order.
